### ISSUE TITLE
Remove cosmic rays

### ISSUE THREAD

**Issue by jradavenport** [jradavenport/spectra#67]
Implement [LA Cosmic](http://www.astro.yale.edu/dokkum/lacosmic/)

Possibly by ingesting the Python implementation...

**Comment by ebellm** [comment]
How about https://github.com/astropy/astroscrappy ?

**Comment by jradavenport** [comment]
Yes! That is probably perfect! Next time I get to play with PyDIS I'll test that out!

**Comment by bmorris3** [comment]
Can confirm – astroscrappy is the way to go. 👍
